pycqed/analysis/analysis_toolbox.py

```diff
@@ -118,47 +118,64 @@ def return_last_n_timestamps(n, contains=''):
 
 
 def latest_data(contains='', older_than=None, newer_than=None, or_equal=False,
```

Comment:
To try to create consistency with other similar functions I would maybe rename the function to "get_latest_data" (same naming convention as get_timestamps_in_range).